pkg/cvo/metrics.go

```diff
@@ -110,7 +117,7 @@ type asyncResult struct {
 // until runContext.Done() and then attempts a clean shutdown
 // limited by shutdownContext.Done().  Assumes runContext.Done()
 // occurs before or simultaneously with shutdownContext.Done().
-func RunMetrics(runContext context.Context, shutdownContext context.Context, listenAddress string, tlsConfig *tls.Config) error {
+func RunMetrics(runContext context.Context, shutdownContext context.Context, listenAddress, certFile, keyFile string, tlsConfig *tls.Config) error {
```

Comment:
might as well drop `tlsConfig` here and [use the cert and key file to construct it internally][1].

[1]: https://github.com/jottofar/cluster-version-operator/blob/543e841ae60248ceb57a15ba70d23b7a61c14e51/pkg/start/start.go#L218

Reply:
Agree. This was first pass just to stand something up quickly.